src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/Models/AssemblyDependencyModel.cs

```diff
@@ -32,14 +32,10 @@ internal class AssemblyDependencyModel : DependencyModel
             IImmutableDictionary<string, string> properties)
             : base(providerType, path, originalItemSpec, flags, resolved, isImplicit, properties)
         {
-            string fusionName = null;
-            if (Properties != null)
-            {
-                Properties.TryGetValue(ResolvedAssemblyReference.FusionNameProperty, out fusionName);
-            }
-
             if (Resolved)
             {
+                string fusionName = null;
```

Comment:
Nit: Unnecessary initialisation

Reply:
It's needed here because the call to `TryGetValue` is conditional.